roles/update/tasks/create-instance.yml

```diff
@@ -0,0 +1,32 @@
+- name: Ensure update log directory exists.
+  ansible.builtin.file:
+    path: "{{ cifmw_update_artifacts_basedir }}"
+    state: directory
+
+- name: create workload launch script
+  template:
+    src: "workload_launch.sh.j2"
+    dest: "{{ cifmw_update_workload_launch_script }}"
+    mode: 0775
+
+- name: create start l3 agent connectivity check scripts
```

Comment:
follow the same convention as otehr role. Name should start with uppercase, no dot at the end